thirdweb/modules/currency.py

```diff
@@ -149,6 +149,16 @@ def burn_from(self, from_address: str, amount: int):
         return self.execute_tx(self.__abi_module.burn_from.build_transaction(
             from_address, amount, self.get_transact_opts()
         ))
+    def transfer(self, to_address: str, amount: int):
+        """ 
+        :param to_address: The address to transfer to
+        :param amount: The amount to transfer
+
+        Transfers the given amount from the current address
+        """
+        return self.execute_tx(self.__abi_module.transfer_from.build_transaction(
```

Comment:
we cannot use transfer_from underneath because on the contract `transfer` and `transferFrom` behaves differently.
1 checks for approval, 1 doesn't.